packages/financial-templates-lib/test/clients/OptimisticOracleClient.js

```diff
@@ -685,6 +705,343 @@ describe("OptimisticOracleClient.js", function () {
       await advanceBlockAndSetTime(web3, currentTime + 1);
       await advanceBlockAndSetTime(web3, currentTime + 2);
 
+      await clientShortLookback.update();
+      result = clientShortLookback.getUnproposedPriceRequests();
+      objectsInArrayInclude(result, []);
+    });
+  });
+  describe("OptimisticOracleV2 contract", function () {
```

Comment:
for all intents and purposes this test (and those in the other implementations) simply copy/paste the existing tests and update them to show they still work with the v2 contrcts.